fe/lib/components/JobCategorySuggest/JobCategorySuggestChoices.tsx

```diff
@@ -68,6 +70,17 @@ const JobCategorySuggestChoices = forwardRef<HTMLInputElement, Props>(
       }
     };
 
+    if (initialValue && selectedJobCategory === undefined) {
+      setSelectedJobCategory('Other');
+
+      const jobCategorySuggest = choices.find(
```

Comment:
Did we want to `onSelect` here? I don't think that's consistent with the other controls taking an initial value. If not, we should be able to set "Other" directly on line 57.